src/promptflow/promptflow/azure/_restclient/flow_service_caller.py

```diff
@@ -393,29 +393,6 @@ def list_connection_specs(
             )
 
 
-    @_request_wrapper()
```

Comment:
Do we need to regenerate rest client?

Reply:
The new client may not be ready since they asked for confirmation today. We could do it latter.